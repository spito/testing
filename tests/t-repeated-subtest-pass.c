#include <cut.h>

TEST(repeated) {
    ASSERT(1);

    REPEATED_SUBTEST("some name", 6) {
        ASSERT(SUBTEST_NO);
    }

    ASSERT(1);
}
