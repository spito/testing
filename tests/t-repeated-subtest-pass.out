[  1] repeated: 6 subtests
    some name #1.............................................................OK
              #2.............................................................OK
              #3.............................................................OK
              #4.............................................................OK
              #5.............................................................OK
              #6.............................................................OK
[  1] repeated (overall).....................................................OK

Summary:
  tests:          1
  succeeded:      1
  filtered out:   0
  suppressed:     0
  skipped:        0
  failed:         0

Points:           0.00
